ramls/patron-block-condition.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Patron block conditions schema",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "A UUID identifying the condition",
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
```

Comment:
Can you use the more strict regexp from https://dev.folio.org/guides/uuids/ ?
Or update https://github.com/folio-org/mod-users/tree/master/ramls raml-util to a more recent version where https://github.com/folio-org/raml/blob/d114de74fcb19767a55d4b724bbde03663442e4f/schemas/uuid.schema exists and use that one?

Reply:
done, thanks